Makefile

```diff
@@ -107,7 +107,7 @@ helm.dependency-update:
 # helm.install: install the local chart into the current kubernetes cluster/namespace
 .PHONY: helm.install
 helm.install: helm.dependency-update
-	helm install $(releaseName) $(chartPath) --render-subchart-notes
+	helm install $(releaseName) $(chartPath) --render-subchart-notes -f $(chartPath)/values.yaml -f $(chartPath)/values-realistic-benchmark.yaml
```

Comment:
:i: -f $(chartPath)/values.yaml This is always done per default, even if you specify a separate one.

Reply:
I figured we most often test with the realistic variant anyway, so we may as well add it. But happy to remove it :shrug: